### DISCUSSION TITLE
Greek Superleague

### DISCUSSION THREAD

**Discussion by tsakmas** [vasqued2/ha-teamtracker#177] [Ideas]
Hello there and  most of all i want to thank you for all the good effort!
Could you consider add more soccer leagues? For me it would be Greek Superleague!

**Comment by vasqued2** [comment]
It's available.  You just have to configure a [custom API](https://github.com/vasqued2/ha-teamtracker?tab=readme-ov-file#sports--leagues-supported-by-configuring-a-custom-api).

For the Greek Superleague
```
sport_path: soccer
league_path: gre.1
```

**Reply by tsakmas** [reply]
Thank you so much !
